pyodide-build/pyodide_build/buildall.py

```diff
@@ -479,6 +479,9 @@ def generate_packages_json(output_dir: Path, pkg_map: dict[str, BasePackage]) ->
                 "imports": [],
                 "file_name": pkg.unvendored_tests.name,
                 "install_dir": pkg.install_dir,
+                "sha256": _generate_package_hash(
```

Comment:
Explained in https://github.com/pyodide/pyodide/pull/2513#issuecomment-1126651792